modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -33,21 +33,26 @@ metadata:
     k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <1>
 spec:
   config: '{
-    "cniVersion": "0.3.0",
-    "name": "a-bridge-network", <2>
-    "type": "cnv-bridge", <3>
-    "bridge": "br0", <4>
-    "isGateway": true,
-    "ipam": {}
-}'
+    "cniVersion": "0.3.1",
+    "plugins": [
+      {
+        "type": "cnv-bridge", <2>
+        "bridge": "br0", <3>
+        "ipam": {}
+      },
+      {
+        "type": "tuning" <4>
+      }
+    ]
+  }'
 ----
 <1> If you add this annotation to your NetworkAttachmentDefinition, your virtual machine instances
 will only run on nodes that have the `br0` bridge connected.
-<2> The `name` value is part of the annotation you will use in the next step.
-<3> The actual name of the Container Network Interface (CNI) plug-in that provides
+<2> The actual name of the Container Network Interface (CNI) plug-in that provides
 the network for this NetworkAttachmentDefinition. Do not change this field unless
 you want to use a different CNI.
-<4> You must substitute the actual name of the bridge, if it is not `br0`.
+<3> You must substitute the actual name of the bridge, if it is not `br0`.
+<4> Required if you want to specify the MAC address of the bridge.
```

Comment:
This is always required. It allows MAC pool manager to assign unique MAC address to the connection.